src/applications/disability-benefits/all-claims/content/addDisabilities.jsx

```diff
@@ -16,8 +16,8 @@ export const addDisabilitiesInstructions = (
     <h4>Examples of conditions</h4>
     <ul>
       <li>Tinnitus (ringing or hissing in ears)</li>
-      <li>PTSD (post-traumatic stress disorder), combat-related</li>
-      <li>Hearing loss, other than tinnitus</li>
+      <li>PTSD (post-traumatic stress disorder)</li>
+      <li>Hearing loss</li>
```

Comment:
These new examples are not in the [autosuggest list](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/disability-benefits/all-claims/content/disabilityLabelsRevised.js#L436). Just checking if that's ok? Users do not have to pick from autosuggest, but it could cause confusion.